### PR TITLE
feat: deprecate responsive type ramp and update ramp to even values

### LINKED ISSUE #1083: Deprecate responsive type ramp for MSFT and update type ramp values
Based on the latest guidance, we should use a flat type ramp instead of a responsive type ramp based on viewports. Viewport/breakpoint support varies and often this created an issues with text reflowing in the middle of viewports.

Instead, guidance will be to use a new instance of the type ramp when you need a change based on viewport.

Updated type sizes:

```
    t1: {
        fontSize: 60,
        lineHeight: 72,
    },
    t2: {
        fontSize: 46,
        lineHeight: 56,
    },
    t3: {
        fontSize: 34,
        lineHeight: 44,
    },
    t4: {
        fontSize: 24,
        lineHeight: 32,
    },
    t5: {
        fontSize: 20,
        lineHeight: 28,
    },
    t6: {
        fontSize: 16,
        lineHeight: 24,
    },
    t7: {
        fontSize: 14,
        lineHeight: 20,
    },
    t8: {
        fontSize: 12,
        lineHeight: 16,
    },
    t9: {
        fontSize: 10,
        lineHeight: 12,
    }
```

### PR DESCRIPTION
**Pull request by chrisdholt**
<body>
closes #1083
<footer>
 

For [details on formatting](https://github.com/Microsoft/fast-dna/wiki/pull-request-guidance) pull requests.

### CODE REVIEW

File: packages/fast-components-styles-msft/src/toggle/index.ts [hunk 1]
```diff
@@ -14,7 +14,7 @@ import {
     ensureContrast,
     toPx,
 } from "@microsoft/fast-jss-utilities";
-import { typeRamp } from "../utilities/typography";
+import { applyTypeRampConfig, typeRamp } from "../utilities/typography";
```
[the remark this answers is not in the document]
Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/toggle/index.ts]: yes, not sure how it snuck back in.

File: packages/fast-components-styles-msft/src/utilities/typography.ts [hunk 2]
```diff
@@ -38,119 +38,70 @@ export interface TypeRamp {
...
+ * @deprecated
  */
 export function applyType(
     typeConfig: keyof TypeRamp,
-    viewport: keyof TypeRampItem
+    viewport?: keyof KeyOfToType<Breakpoints, TypeRampItemConfig>
 ): CSSRules<DesignSystem> {
+    if (viewport) {
+        /* tslint:disable-next-line:no-console */
+        console.warn(
```
[the remark this answers is not in the document]
Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/utilities/typography.ts]: I think we need general documentation to let them know that when deprecating things we will add console warnings which they can remove using tooling such as webpack. I don't think we should check for prod or dev env variables.

File: packages/fast-components-styles-msft/src/utilities/typography.ts [hunk 3]
```diff
@@ -38,119 +38,70 @@ export interface TypeRamp {
...
-            lineHeight: 20,
-        },
+        fontSize: 10,
+        lineHeight: 12,
     },
 };
 
 /**
  * Applies a type ramp config instance based on viewport
+ * @deprecated
```
Comment (nicholasrice) [inline packages/fast-components-styles-msft/src/utilities/typography.ts]: maybe add a note that only the second argument is deprecated, not the whole function.

Comment (chrisdholt) [reply packages/fast-components-styles-msft/src/utilities/typography.ts]: @nicholasrice I intentionally deprecated this function entirely. Now that the type ramp is flat, the return value of this function matches applyTypeRampConfig. It makes the most sense to me to only keep one of the functions.

**Pull request merged**